CONTRIBUTING.md

```diff
@@ -40,6 +40,16 @@ Configuration is available for cmake, [documented elsewhere](https://cmake.org/d
 
 Mac support is pretty early and while it will compile, it's not usable for debugging yet. Builds happen with cmake the same way as Linux.
 
+### Android
+
+To build the components required to debug an Android target, first gather everything from [Dependencies](#dependencies) below, then run:
+```
+mkdir build-android
+cd build-android
+cmake -DBUILD_ANDROID=On -DANDROID_ABI=armeabi-v7a -DANDROID_NATIVE_API_LEVEL=23 ..
+make
+```
```

Comment:
Thanks. I don't know about anyone else, but at least I'm building it on windows for the nightly builds. I imagine there are plenty of windows devs targeting android just because there are so many windows devs in general.

I think it's fine to expect the android subset of users to have bash available though on windows, since cmd is so awful.